config/cmake_ext_mod/ConfigureChecks.cmake

```diff
@@ -142,19 +137,10 @@ if (CYGWIN)
   set (${HDF_PREFIX}_HAVE_LSEEK64 0)
 endif ()
 
-#-----------------------------------------------------------------------------
-#  Check for the math library "m"
-#-----------------------------------------------------------------------------
 if (MINGW OR NOT WINDOWS)
   CHECK_LIBRARY_EXISTS_CONCAT ("m" ceil     ${HDF_PREFIX}_HAVE_LIBM)
-  CHECK_LIBRARY_EXISTS_CONCAT ("dl" dlopen     ${HDF_PREFIX}_HAVE_LIBDL)
```

Comment:
Does mingw not need these? Mingw generates windows objects on linux, right?

Reply:
Yes, but there is no `libdl` in MinGW as it uses `LoadLibrary`. Maybe you're thinking of Cygwin?